samples/HttpClientFactorySample/Program.cs

```diff
@@ -19,19 +19,23 @@ public class Program
         public static async Task Run()
         {
             var services = new ServiceCollection()
-                .AddHttpClient()
+                .AddHttpClient("github", c =>
+                {
+                    c.BaseAddress = new Uri("https://api.github.com/");
+
+                    c.DefaultRequestHeaders.Add("Accept", "application/vnd.github.v3+json"); // Github API versioning
+                    c.DefaultRequestHeaders.Add("User-Agent", "HttpClientFactory-Sample"); // Github requires a user-agent
+                })
```

Comment:
I think that most of the 'common' stuff we want is right on the `HttpClient` - until we start talking about 3rd party libraries